mautic/mautic-documentation
# Implement Vale Linter with GitHub Actions

This PR will enable the [Vale Linter](https://github.com/errata-ai/vale) on the docs.

This will check every PR for errors (typos, major grammar issues) and make suggestions.

We are starting with the [Google Style Guide](https://github.com/errata-ai/Google) and we can refine over time.

All hard errors have been addressed but there are a lot of suggestions which we can work on over time.

Big thanks to @swati-thacker13 for all your work on this as part of the Season of Docs!

## Testing this PR

1. [Install Vale](https://docs.errata.ai/vale/install)
2. Clone down the Docs repo
2. Check out this pr using `gh pr checkout 147`
3. Run Vale using vale <file>
